js_modules/dagit/packages/core/src/app/PythonErrorFragment.tsx

```diff
@@ -0,0 +1,13 @@
+import {gql} from '@apollo/client';
+
+export const PYTHON_ERROR_FRAGMENT = gql`
+  fragment PythonErrorFragment on PythonError {
+    __typename
+    message
+    stack
+    cause {
+      message
+      stack
+    }
+  }
+`;
```

Comment:
yeah, I didn't want the initial PR to have a ton of changes so that its easier to review but I'll do what I did in the original PR where I moved the dependencies around and changed all the imports so that theres only 1 definition